src/content/nft/index.md

```diff
@@ -77,6 +77,19 @@ This website has an alternative domain name powered by NFTs, **ethereum.eth**. O
 
 ## How do NFTs work? {#how-nfts-work}
 
+Thinking about this from a very high level, there are many blockchains that support NFTs but you will find majority under the Ethereum blockchain. There is a difference between NFT tokens and Ethereum tokens in that NFT tokens have extra information stored in them. This extra information is what really makes an NFT an NFT! NFTs are unique crypto tokens that are maganed on a blockchain and the blockchain acts as the decentralized ledger that tracks the ownership and transaction history of each NFT. An NFT is created, or in the proper terms "minted" from digital objects that represent many things. For example, an NFT can consist of items such as:
+
+- GIFs
+- Collectibles
+- Music
+- Videos
```

Comment:
These all fit into the category of digital art. Would consider other examples, ie: as deeds to real world items (ie a car or physical collectible), as your ticket to a real world event, tokenized invoices that can be uses as collateral, legal documents and signatures. Lot of options =)